index.html

```diff
@@ -38,5 +38,95 @@ <h1> <span class="wrapper">I'm Mariane</span> <span class="wrapper">Glad to see<
             </ul>
          </div>
     </section>
+    <main>
+        <section class="card-1">
+            <div class="card-1-bg"></div>
+            <h2>Tonic</h2>
+            <div class="canopy"></div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="lang-btn">
+               <ul>
+                  <li><a href="#"><button>html</button></a></li>
+                  <li><a href="#"><button>css</button></a></li>
+                  <li><a href="#"><button>javaScript</button></a></li>
+               </ul>
+            </div>
+            <div class="card-1-btn"><button>See Project</button></div>
+        </section>
+        <section class="card-2">
+            <div class="card-2-bg"></div>
+            <h2>Tonic</h2>
+            <div class="canopy"></div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="lang-btn">
+               <ul>
+                  <li><a href="#"><button>html</button></a></li>
+                  <li><a href="#"><button>css</button></a></li>
+                  <li><a href="#"><button>javaScript</button></a></li>
+               </ul>
+            </div>
+            <div class="card-1-btn"><button>See Project</button></div>
+        </section>
+        <section class="card-3">
+            <div class="card-3-bg"></div>
+            <h2>Tonic</h2>
+            <div class="canopy"></div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="lang-btn">
+               <ul>
+                  <li><a href="#"><button>html</button></a></li>
+                  <li><a href="#"><button>css</button></a></li>
+                  <li><a href="#"><button>javaScript</button></a></li>
+               </ul>
+            </div>
+            <div class="card-1-btn"><button>See Project</button></div>
+        </section>
+        <section class="card-4">
+            <div class="card-4-bg"></div>
+            <h2>Tonic</h2>
+            <div class="canopy"></div>
```

Comment:
- [ ] For good semantics reasons and as required for the project, kindly make sure the the container of the section is a `section` tag and not `main` tag